lib/elixir/lib/map_set.ex

```diff
@@ -398,6 +398,7 @@ defmodule MapSet do
     import Inspect.Algebra
 
     def inspect(map_set, opts) do
+      opts = %Inspect.Opts{opts | charlists: :as_lists}
```

Comment:
I think it's better default but can't help but wonder if folks would ever prefer to opt-in to the previous behaviour. E.g. they do `inspect(map_set, chartists: :as_charlists)` and now the option is ignored